application/event_processor/changes.py

```diff
@@ -65,7 +65,7 @@ def update_changes_with_address(changes: dict, dos_service: DoSService, nhs_uk_e
     dos_address = dos_service.address
 
     if dos_address != nhs_uk_address_string:
-        logger.debug(f"Address is not equal, {dos_address=} != {nhs_uk_address_string=}")
+        logger.info(f"Address is not equal, {dos_address=} != {nhs_uk_address_string=}")
```

Comment:
Why is address info level but all others are debug?

Reply:
yeah prob should stay as debug and instead make debug logs on in lower environments
main annoyance was that i couldnt see these logs when i needed to as the log level was set to info